http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1211,9 +1211,9 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       ServiceAccountName(namespaceAttributes.getKubernetesServiceAccount)
     )
 
-  private[util] def deleteWsmNamespaceResource(workspaceId: WorkspaceId,
-                                               app: App,
-                                               wsmResource: AppControlledResourceRecord
+  private[util] def deleteAndPollWsmNamespaceResource(workspaceId: WorkspaceId,
```

Comment:
Nit, but this function deletes more than just the namespace, would it make sense to rename it to something like `deleteAndPollWsmAppResources`?

Reply:
... this is aligning with the changes to naming in the method, too

Reply:
good point, I forgot to change the naming here!